### ISSUE TITLE
Grief After and During war events

### ISSUE THREAD

**Issue by TitanKaivor** [TownyAdvanced/Towny#3577]
### What steps will reproduce the problem?
1. Players are allowed to destroy and build blocks during and after war event even the line is set false:
```
# This setting SHOULD NOT BE USED unless you want the most chaotic war possible.
# The editable_materials list in the Warzone Block Permission section should be used instead.
    allow_block_griefing: 'false'
```

### What is the expected output?
1. Players shouldn't be able to destroy or build in enemy towns.

### What is your Towny version?
0.95.2.2

### What is your spigot/craftbukkit version?
Spigot 1.15

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. There are no log errors.
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/76ttwaDc
3. Your townyperms.yml : https://pastebin.com/Kkr0Bph9

**Comment by Goosius1** [comment]
Hi there, I think this setting is only used to turn griefing ON, not turn it off.

If you want to turn block griefing off, try modifying this section in your config yml:

'''
  warzone:

    # List of materials that can be modified in a warzone.

    # '*' = Allow all materials.

    # Prepend a '-' in front of a material to remove it. Used in conjunction with when you use '*'.

    # Eg: '*,-chest,-furnace'

    editable_materials: tnt,oak_fence,birch_fence,spruce_fence,jungle_fence,dark_oak_fence,acacia_fence,ladder,oak_door,birch_door,spruce_door,jungle_door,dark_oak_door,acacia_fence,iron_door,fire

    item_use: 'true'
'''

**Comment by Goosius1** [comment]
Which types of blocks are you seeing being destroyed and built on your server?

**Comment by Goosius1** [comment]
Also the timing seems important to clarify:

* Are the blocks griefable before the event starts?
* Are the blocks griefable after the event finishes?

**Comment by LlmDl** [comment]
These parts of the form were skipped:

Startup
Permission file from external permission plugin

Please provide them.

**Comment by TitanKaivor** [comment]
> 
> 
> Which types of blocks are you seeing being destroyed and built on your server?

Every type of block, even open chests.

**Comment by TitanKaivor** [comment]
> 
> 
> Also the timing seems important to clarify:
> 
>     * Are the blocks griefable before the event starts?
> 
>     * Are the blocks griefable after the event finishes?

They are not griefable before the event starts but they are griefable during and after the event if a nation has enemy another nation.

**Comment by TitanKaivor** [comment]
> 
> 
> These parts of the form were skipped:
> 
> Startup
> Permission file from external permission plugin
> 
> Please provide them.

PermissionsEx: https://pastebin.com/qfQs2zTE
I deleted nicknames, I don't think it's fault of permissions plugin

**Comment by LlmDl** [comment]
You need to remove all of the nodes from Towny in your non-admin/non-moderator ranks.

All of those nodes are already given out correctly in the townyperms.yml file.

You still need to show us your startup latest.log

**Comment by TitanKaivor** [comment]
> 
> 
> You need to remove all of the nodes from Towny in your non-admin/non-moderator ranks.
> 
> All of those nodes are already given out correctly in the townyperms.yml file.
> 
> You still need to show us your startup latest.log
Towny nodes removed from permissionsex.
Log: https://pastebin.com/Mv19J5yr

**Comment by TitanKaivor** [comment]
> 
> 
> You need to remove all of the nodes from Towny in your non-admin/non-moderator ranks.
> 
> All of those nodes are already given out correctly in the townyperms.yml file.
> 
> You still need to show us your startup latest.log

Towny nodes removed from permissionsex.
Log: https://pastebin.com/Mv19J5yr

**Comment by LlmDl** [comment]
This is fixed as of 0.95.2.3, you can try it out for yourself with this advanced copy:
https://cdn.discordapp.com/attachments/388338495036456960/658718976074842130/Towny-0.95.2.3.jar
